db/db_flush_test.cc

```diff
@@ -107,7 +107,7 @@ TEST_F(DBFlushTest, SyncFail) {
   TEST_SYNC_POINT("DBFlushTest::SyncFail:2");
   fault_injection_env->SetFilesystemActive(true);
   // Now the background job will do the flush; wait for it.
-  dbfull()->TEST_WaitForFlushMemTable();
+  dbfull()->TEST_WaitForFlushMemTable().PermitUncheckedError();
```

Comment:
Why not an ASSERT_OK?  If it is not consistent, please add a comment as to why to ignore it rather than check it...